src/vm/codeman.cpp

```diff
@@ -2186,19 +2189,36 @@ HeapList* EEJitManager::NewCodeHeap(CodeHeapRequestInfo *pInfo, DomainCodeHeapLi
     } CONTRACT_END;
 
     size_t initialRequestSize = pInfo->getRequestSize();
-    size_t minReserveSize = VIRTUAL_ALLOC_RESERVE_GRANULARITY; //     ( 64 KB)           
+#if defined(FEATURE_JIT_PITCHING)
+    size_t minReserveSize = 0;
+    if (CLRConfig::GetConfigValue(CLRConfig::INTERNAL_JitPitchEnabled) != 0)
+        minReserveSize = OS_PAGE_SIZE; //     ( 4 KB)
+    else
+        minReserveSize = VIRTUAL_ALLOC_RESERVE_GRANULARITY; //     ( 64 KB)
+#else
+    size_t minReserveSize = VIRTUAL_ALLOC_RESERVE_GRANULARITY; //     ( 64 KB)
+#endif
 
```

Comment:
are these size adjustments necessary now that you are using HostCodeHeap? The HostCodeHeap internally uses a free list to re-use space that was previously allocated.